docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+   forwarding the port. In most cases that'll just mean pressing ``CTRL`` + ``C``
+   in the terminal where the port-forward command is running.
+
+Configuration
+-------------
+
+The helm chart uses the :doc:`Trino container image </installation/containers>`
+by default. The Docker image already contains a default configuration to get
+started, and some catalogs to allow you to explore Trino. Kubernetes allows you
+to mimic a :doc:`traditional deployment </installation/deployment>` by updating
+the ``values.yaml`` file. It's important to understand how files such as the
+Trino configuration, JVM, and various :doc:`catalog properties </connector>` are
+configured in Trino before updating the values.
+
+.. _creating-your-own-values-yaml:
+
+Creating your own values.yaml
+"""""""""""""""""""""""""""""
+
+When you use your own ``values.yaml``, you only override the values you specify.
+The remaining properties use their default values. Add a ``values.yaml`` with
+the following configuration:
+
+.. code-block:: yaml
+
+   image:
+     tag: "|trino_version|"
+   server:
+     workers: 2
+     config:
+       memory:
+         heapHeadroomPerNode: "1GB"
+       query:
+         maxMemory: "4GB"
+         maxMemoryPerNode: "1GB"
+   coordinator:
+     jvm:
+       maxHeapSize: "8G"
+   worker:
+     jvm:
+       maxHeapSize: "8G"
+
+These values are higher than the defaults and allow Trino to use more memory
+and run more demanding queries. If the values are too high, Kubernetes might
+not be able to schedule some Trino pods, depending on other applications
+deployed in this cluster and the size of the cluster nodes.
+
+#. ``.image.tag`` is set to the current version, |trino_version|. Set
+   this value if you need to use a specific version of Trino. The default is
+   ``latest``.
+#. ``.server.workers`` is set to ``2``. This value sets the number of
+   workers, in this case, a coordinator and two worker nodes are deployed.
+#. ``.server.config.memory.heapHeadroomPerNode`` is set to ``1GB``.
+   This sets the :ref:`prop-resource-memory-heap-headroom-per-node` property.
+#. ``.server.config.query.maxMemory`` is set to ``4GB``.
+   This sets the :ref:`prop-resource-query-max-memory` property.
+#. ``.server.config.query.maxMemoryPerNode`` is set to ``1GB``.
+   This sets the :ref:`prop-resource-query-max-memory-per-node` property.
+#. ``.coordinator.jvm.maxHeapSize`` is set to ``8GB``.
+   This sets the maximum heap size in the JVM of the coordinator. See
+   :ref:`jvm_config`.
+#. ``.worker.jvm.maxHeapSize`` is set to ``8GB``.
+   This sets the maximum heap size in the JVM of the worker. See
+   :ref:`jvm_config`.
+
+   .. warning::
+      These memory settings need to be tuned carefully as setting some values
+      outside of the range of the maximum heap size will cause Trino startup to
+      fail. See the warnings listed on :doc:`/admin/properties-resource-management`.
+
+The ``values.yaml`` has
```

Comment:
Not sure what you want to say with this sentence .

Reply:
just want to refer to the properties list on the chart, fixed wording.